main/.editorconfig

```diff
@@ -6,4 +6,4 @@ end_of_line = lf
 indent_size = 4
 indent_style = space
 insert_final_newline = true
-max_line_length = 120
```

Comment:
What was the reason for the change?

I personally have no strong preference on this.

Reply:
That's the default in the sonar-developer-toolset. There's no reason to use a different number here.